ansible/roles/dashmate/templates/dashmate.json.j2

```diff
@@ -213,7 +224,25 @@
             },
             "consensus": {
               "createEmptyBlocks": true,
-              "createEmptyBlocksInterval": "3m"
+              "createEmptyBlocksInterval": "3m",
+              "peer": {
+                "gossipSleepDuration": "{{dashmate_platform_drive_tenderdash_consensus_peer_gossip_sleep_duration}}",
+                "queryMaj23SleepDuration": "{{dashmate_platform_drive_tenderdash_consensus_peer_query_maj23_sleep_duration}}"
+              },
+              "unsafeOverride": {
```

Comment:
Why not using consensus params instead of overrides?

Reply:
We can't change the consensus param without resetting the network. Having these options we can experiment with numbers quickly. It's not for persistent configuration.

Reply:
Acually we can chance the consensus params once an hour on testnet.

Reply:
But I agree this is faster for now.

Reply:
You mean if we hardcode this change versions?